spec/components/provider_interface/conditions_component_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::ConditionsComponent do
+  describe 'rendered component' do
+    let(:conditions) { ['Fitness to teach check'] }
+
+    it 'renders the conditions' do
+      application_with_conditions_met = build(:application_choice, status: 'recruited', offer: { 'conditions' => conditions })
+      result = render_inline(described_class.new(application_choice: application_with_conditions_met))
+
+      expect(result.to_html).to include('Fitness to teach check')
+    end
+
+    it 'indicates whether conditions are met' do
```

Comment:
How about one more scenario, 'indicates whether conditions are pending'? For non-deferred offers.